src/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Components using the react-intl module require access to the intl context.
+ * This is not available when mounting single components in Enzyme.
+ * These helper functions aim to address that and wrap a valid,
+ * English-locale intl context around them.
+ */
+import { cloneElement } from 'react';
+import { IntlProvider, intlShape } from 'react-intl';
+import { mount, shallow, render } from 'enzyme';
+
+export const mockIntl = {
```

Comment:
Should we be using the mockIntl object in shallow/render/mount?

Reply:
We could. It's essentially doing the same thing as the message proxy below, but not using the "real" `IntlProvider` API.

If we used that instead, we run the risk of not being in sync with the real API, I suppose, but we could potentially drop the need for `react-intl` as a dev/peer dependency altogether.

Reply:
I lean towards using the "real" IntlProvider API.